.gitignore

```diff
@@ -0,0 +1,5 @@
+<<<<<<< HEAD
```

Comment:
Please fix this merge comments.

Reply:
Yes sure, I have fixed the merge comments.